Capstone/Detail View/DetailsViewController.h

```diff
@@ -7,7 +7,9 @@
 #import "UIKit/UIKit.h"
 #import "College.h"
 NS_ASSUME_NONNULL_BEGIN
-@interface DetailsViewController : UIViewController
+@interface DetailsViewController : UIViewController<CLLocationManagerDelegate, MKMapViewDelegate> {
+    CLLocationManager *locationManager;
```

Comment:
let's move this — and then interface — to the .m file, as `locationManager` should be private. you should be able to conform to the delegate protocols & the variable on `@interface DetailsViewController` in the .m file

Reply:
My bad. I rushed this PR as I didn't make much progress on the Translation branch. 
Resolved!